### PR TITLE
Bug 1917658: add tech preview badge for camel K operator instances

### PR DESCRIPTION
**Pull request by sahil143**
**Fixes**: https://issues.redhat.com/browse/ODC-5346

**Analysis / Root cause**: missing tech preview badge camel K operator instance 

**Solution Description**: added tech preview badge

**Screen shots / Gifs for design review**: 

Screenshot 2021-01-19 at 10 38 41 AM
Screenshot 2021-01-19 at 10 41 46 AM
Screenshot 2021-01-19 at 10 41 58 AM
Screenshot 2021-01-19 at 10 42 14 AM

**Unit test coverage report**: 

**Test setup:**

**Browser conformance**: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/knative-plugin/src/models.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
   STRIMZI_KAFKA_APIGROUP,
   CAMEL_APIGROUP,
 } from './const';
+import { BadgeType } from '@console/shared/src/components/badges/badge-factory';
```
Comment (invincibleJai) [inline frontend/packages/knative-plugin/src/models.ts:13]: nit: import order

**Review by invincibleJai — COMMENTED** [review]
All looks good @sahil143 , thanks!!

**Review by bgliwa01 — APPROVED** [review]
lgtm!

**Pull request merged**